velox/py/runner/PyLocalRunner.cpp

```diff
@@ -36,12 +37,50 @@ std::mutex& taskRegistryLock() {
   return lock;
 }
 
+std::unordered_set<std::string>& connectorRegistry() {
+  static std::unordered_set<std::string> registry;
+  return registry;
+}
+
 } // namespace
 
 namespace py = pybind11;
 
+void registerHive(const std::string& connectorId) {
+  connector::registerConnectorFactory(
+      std::make_shared<connector::hive::HiveConnectorFactory>());
+
+  // TODO: Allow Python users to specify connector configs.
+  std::unordered_map<std::string, std::string> configValues = {};
+  const auto& configs =
+      std::make_shared<velox::config::ConfigBase>(std::move(configValues));
```

Comment:
Is this making a reference to a temporary value?  I'm surprised this works

Reply:
TIL https://herbsutter.com/2008/01/01/gotw-88-a-candidate-for-the-most-important-const/

Reply:
Ugh, lucky const!